config/middleware.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const cors = require("cors");
+const helmet = require("helmet");
+const session = require("express-session");
+const sessionConfig = require("./sessionConfig");
+
+module.exports = server => {
+  const protected = (req, res, next) => {
```

Comment:
Your protected function looks great.  I like that you threw in some regex.  Great job